front-end/Dockerfile

```diff
@@ -0,0 +1,21 @@
+
+
+FROM node:latest as build-step
+
+RUN mkdir -p /app
+
+WORKDIR /app
+
+COPY package*.json /app
+
+RUN npm install
+
+COPY . /app
+
+RUN npm run build --prod
+
+ 
+
+FROM nginx:1.25.3
```

Comment:
I like that you gave a name for the build step. Given that, you should give this one a name as well to be consistent